deps/obs-scripting/obs-scripting-python.c

```diff
@@ -17,7 +17,6 @@
 ******************************************************************************/
 
 #include "obs-scripting-python.h"
-#include "obs-scripting-config.h"
```

Comment:
Would the various obs-scripting code file changes (*.c, *.h.in, *.i) be too difficult to extract from this PR?

Reply:
That particular change is directly tied into the change of the CMake file, it's not a logically "separate" change that should live in an independent PR.